zhaquirks/xiaomi/__init__.py

```diff
@@ -414,6 +419,16 @@ class MotionCluster(LocalDataCluster, MotionOnEvent):
     reset_s: int = 70
 
 
+class DeviceTemperatureCluster(LocalDataCluster, DeviceTemperature):
+    """Device Temperature Cluster."""
+
+    cluster_id = DeviceTemperature.cluster_id
+
+    def update_attribute(self, attrid, value):
+        """attribute value reported"""
```

Comment:
"""Attribute value reported."""

Reply:
That's not the main problem why is pipeline failing

Reply:
The py test is failing in other files that you have **not** making changes in and that test its broken or some other have doing bad commitments.

"Unchanged files with check annotations **Beta**"